modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-instance/src/main/java/com/liferay/dynamic/data/mapping/data/provider/instance/DDMStorageTypesDataProvider.java

```diff
@@ -43,11 +48,31 @@ public DDMDataProviderResponse getData(
 
 		List<KeyValuePair> keyValuePairs = new ArrayList<>();
 
+		Optional<HttpServletRequest> httpServletRequestOptional =
+			ddmDataProviderRequest.getParameterOptional(
+				"httpServletRequest", HttpServletRequest.class);
+
+		HttpServletRequest httpServletRequest =
+			httpServletRequestOptional.orElse(null);
+
 		Set<String> storageTypes =
 			ddmStorageAdapterTracker.getDDMStorageAdapterTypes();
```

Comment:
@carolmariaabb, as used rule. The variables used first should be declared first. Can you please fix?